react/index.js

```diff
@@ -244,23 +261,24 @@ export default class Footer extends Component {
             />
           </div>
         </div>
-        <div className="vtex-footer__container flex justify-between bg-white mid-gray">
+        <div className="vtex-footer__container pv5-s flex justify-between ph4-s bg-white mid-gray">
           <FooterPaymentFormMatrix
             paymentForms={paymentForms}
             horizontal
             showInColor={showPaymentFormsInColor}
           />
         </div>
-        <div className="vtex-footer__container flex justify-between bg-white mid-gray">
-          <div className="vtex-footer__text-container w-100-s w-80-ns flex flex-wrap">
+        <div className="vtex-footer__container pt5-s flex justify-between bg-white mid-gray ph4-s">
+          <div className="vtex-footer__text-container w-100-s pb5-s w-80-ns flex flex-wrap">
             {storeInformations &&
               storeInformations.map(({ storeInformation }, index) => (
-                <div
+                <p
```

Comment:
Why?

Reply:
Because this is rendenring a paragraph, so it should be using a `p` element

Reply:
Nice! It was a test. You passed.